HACKING.md

```diff
@@ -0,0 +1,13 @@
+# Hacking
+
+## Build documentation
+
+Build and open in a Web browser:
+
+```sh
+$ cd doc
+$ make html
```

Comment:
I've got a warning for Sphinx 4.3.0
```
/home/georgymoiseev/Development/github/luafun/luafun/doc/reference.rst:4: WARNING: duplicate object description of fun, other instance in under_the_hood, use :noindex: for one of them
```

Reply:
Cannot reproduce your result. I have Sphinx 4.3.0 installed from the main Gentoo tree. However I tried to install it using pip and there is no difference: still no warning.

Even if we'll look at the files:

```sh
$ grep -R module.*fun reference.rst under_the_hood.rst
reference.rst:.. module:: fun
under_the_hood.rst:.. currentmodule:: fun
```

We use `currentmodule` in `under_the_hood.rst`, not `module` (see the commit message regarding this). Don't know where the directive is documented, but can link the [changelog entry](https://github.com/sphinx-doc/sphinx/blob/17dfa811078205bd415700361e97e945112b89eb/CHANGES.old#L1100-L1101), where it was introduced.

So, it seems, we should have no warning. Or should have warnings regarding all those `currentmodule` directives in 10 files. One warning is strange.

Please, try to remove the `_build` directory entirely and repeat.

Reply:
> Please, try to remove the _build directory entirely and repeat.
Yeah, it fixed the issue. Seems like it was some local bug.